### DISCUSSION TITLE
Crafting a coherent story and lore:

### DISCUSSION THREAD

**Discussion by DokimiCU** [jeremyshannon/Exile#54] [General]
To get narrative elements right it would be helpful to pull Exile's improvised and disconnected lore into a coherent story.

I like leaving things open to interpretation (at least as communicated to the player), but the basic facts need to be figured out so that it actually makes sense.

- The current day civilizations: who are they? where are they? why is their technology so primitive? Why are they so diverse, including even green people?
- Universal practice of exiling people to this place: why do they all do this? Why is it so similar despite vast cultural differences?
- Transport: how are people sent here despite the primitiveness of the sending cultures?
- The Land: where is this? Why are their volcanoes everywhere? Why is the weather so crazy? Why is it so barren? Who, if anyone, has political control here?
- The City: why is it here? Why is it underground? Why is the tech so advanced?
- The Catastrophe: what happened to depopulate the city? 
- The question of the age: what is this world wrestling with? What can they player do that is meaningful in this world?

**Comment by DokimiCU** [comment]
Stuff that's already in the game:
- Super diverse origin civilizations, with differently languages, cultures, politics etc
- Universal, almost standardized, practice of exiling people here
- The land's name: no one has the exactly the same name for this place
- The mythic_terror = {
  "Great Calamity",
  "Unending Curse",
  "Ancient Curse",
  "Curse",
  "Catastrophe",
  "Great Dying",
  "Mythic Terror",
  "Night",
  "Darkness",
  "Scourge",
  "Sleeping Evil",
  "Firey Breath",
  "Great Burning",
  "Great Bleeding",
  "Ancient Exodus",
  "Fearful Horror",
  "Evil Spirits",
  "Ghosts",
  "Haunting",
  "Rumbling Earth",
  "Burning Rock",
  "Great Folly",
  "Old Tales",
  "Evil Wind",
  "Bitter Waters",
  "Howling Dust",
  "Hateful Sky",
  "Great Confusion",
  "Lost"
}
- Land of the Ancients
- You got pushed through the "Gateway"
- The City itself. A megacity that probably could house millions. With super-advanced tech. But underground.
- Destruction of the city. The whole thing is crumbling, being consumed by lava.

**Comment by wizardofgcc** [comment]
This is definitely something that should be realised to form a complete atmosphere of Exile, however, in my opinion, too much focus shouldn't be put on where the player character came from. From what I see from the game right now, it should be about the exile, and enduring it, rather than what came before it. The past should probably should be presented in the game as small bits, scattered around the world, that the player has to interpret to figure out the past.

My idea of what happened, at least what I imagine based on implemented content:
The civilization that resided in the world of Exile was one the first advanced civilizations, however they significantly damaged their planet somehow (e. g. by causing a major climate change), to such a degree that the surface became mostly uninhabitable. To survive, they constructed the enormous city underground. Later, attempts to fix the surface were made: artificial species were introduced, weather was changed in someway, thus the stale and weird world. Later when the meta-stim was invented, which made everyone a god. That quite obviously caused a war, which killed most of the people. The remaining population united and focused on resetting the dire state of events in their world by spreading members of their civilization to other planets or universes. Since they had the teleportation machines that we have in the game, this was probably done by a much more powerful version. They would create a gateway to wherever, send a group of people modified for the conditions of the place in (thus explaining the varying appearance; maybe the green skin helps with UV radiation?), and move on to another place. The people sent there had little idea of the advanced technologies, maybe just a vague idea that they came from another world and that it was destroyed by a disaster, so they were basically reset to cavemen. The project for this eventually finished, and the machine(s) made nonoperational, however in the world they sent the people to the "gateways" remained, which would be able to send people back to the main world. The civilizations created eventually developed to the iron age, and found a use for those old gateways to exile people. You are one of those sent through the gateways, left to endure this world.

**Comment by jeremyshannon** [comment]
Wizard: You've mirrored a lot of my ideas on the subject. I envision a long series of catastrophes, starting with an abortive attempt to control the weather which made the surface unlivable.

At least some of the rare skin colors predate the fall IMO, and were modfications for fashion or for particular castes, especially during the breakup of the Ancients' civilization, there was a lot of increasing division and even civil wars. Most such modifications died with the Ancients' cities, but a handful breed true and have survived. Having one or two be modifications made to allow emigrants to survive in whatever place they were fleeing to sounds good to me though.
